dune-project

```diff
@@ -1,26 +1,5 @@
 (lang dune 3.0)
-(name notty)
+(name notty-community)
 
 (formatting disabled)
-(generate_opam_files true)
```

Comment:
What's the benefit of removing this?

Reply:
this is personal preference on account of this feature being devoid of value past the initial generation, duplicating all the informations over two files and making it harder to contribute changes to the opam file.